sre_maturity.md

```diff
@@ -0,0 +1,115 @@
+# SRE Maturity Model
+
+## Phase 1: The ‘Crawl’ Phase
+
+### Overview
+
+You’re starting from zero, with a net new team.
+You may only have a small SRE team, or an SRE function within your development team.
+You and your team are technically capable of surviving.
+You are probably providing a ‘white glove’ experience to your users at this stage.
+
+### Characteristics
+
+#### People and Culture
+
+You have a strong SRE voice in your team, someone who advocates for reliability features.
+They establish people focused mantras like "It's OK to fail", "Hope is not a strategy" and "Trust by default & extend on it".
+Communication is important in your team and you hire diverse people based on soft skills & leadership rather than specific technologies.
+
+At this phase, having buy-in from executive level leadership is essential.
+Without this it will be very difficult to get their support later, and eventually their engagement.
+
+The product manager for you service should have a good understanding of SRE.
+They should help you advocate for service features that make the service more reliable.
+
+#### Observability
+
+The foundations for monitoring and observing your service are in place.
+Being able to measure something meaningful about your service is important, and supplements any feedback you get from initial users.
+A common way to do this is via metrics exposed by your service.
+
+You have a simple Service Level Objective (SLO) for you service to start out with.
+It doesn't have to be completely representative of what users expect from your service, but it forms the building blocks of an SLO review process and improving on them in time.
+Starting with a lower SLO than where you want to get to sets expectations early.
```

Comment:
Here we might mention a "graduation" process from "informing" to "enforcing" SLO.  This might take a few sprints.

Reply:
```suggestion
Starting with a lower SLO than where you want to get to sets expectations early. There's a graduation process with SLOs from "informing" to "enforcing". You can use SLOs to inform decisions in this early stage. As they get refined and wider awareness & support, you can think about enforcement of an error budget policy.
```
@jeremyeder This sound resonable?